src/System Application/App/app.json

```diff
@@ -36,10 +36,13 @@
     "TranslationFile",
     "GenerateCaptions"
   ],
+  "resourceFolders": [
+    ".resources"
```

Comment:
Are you using .ressource because ressources is not possible? At least that's what I've discovered. Dot folders are currently always for content that is not repository relevant. Like .alpackages, .vscode, .snapshots. I would prefer data here.